src/styles/bloks/card.scss

```diff
@@ -0,0 +1,100 @@
+@use '../utils/variables' as *;
+
+body {
+  margin: 0;
+}
+
+.card {
+  width: $card-width;
+  height: $card-height;
+  background-color: white;
+  box-sizing: border-box;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  &__photo {
+    width: 160px;
+    height: 134px;
+    margin-top: 32px;
+    margin-left: 19px;
+    object-fit: fill;
+  }
+
+  &__description {
+    margin-left: $card-margin;
+    margin-right: $card-margin;
+    font-size: 12px;
+    text-align: left;
+    margin-top: 36px;
+    font-family: $main-font;
+    font-weight: 500;
+    line-height: 18px;
+    color: #060b35;
+  }
+
+  &__code {
+    font-family: $main-font;
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+    margin-left: $card-margin;
+    margin-top: 4px;
+    color: #616070;
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-left: $card-margin;
+    margin-right: $card-margin;
+    margin-top: 23px;
+
+    &_1 {
+      font-family: $main-font;
+      font-size: 12px;
+      font-weight: 400;
+      color: #616070;
+    }
+
+    &_2 {
+      font-family: $main-font;
+      font-size: 16px;
+      font-weight: 700;
+      color: #060b35;
+    }
+  }
+
+  &__buy {
+    width: 166px;
+    height: 40px;
+    margin-left: $card-margin;
```

Comment:
The variable `$card-margin` is used here. Ensure it is defined in the `variables` file to prevent compilation errors.